provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/Provisio.java

```diff
@@ -95,8 +98,13 @@ public List<Runtime> findDescriptorsForPackagingTypeInExtensionRealms(MavenProje
 
   public Runtime parseDescriptor(InputStream inputStream, MavenProject project) {
```

Comment:
Maybe: 
```
public static Properties getPropertiesWithSystemOverrides(MavenProject project) {
    Properties properties = new Properties(project.getProperties());
    properties.putAll(System.getProperties());
    return properties;
  }

  public Runtime parseDescriptor(InputStream inputStream, MavenProject project) {
    RuntimeReader parser = new RuntimeReader(Actions.defaultActionDescriptors(), versionMap(project));
    Map<String, String> variables = Maps.newHashMap();
    variables.putAll((Map) getPropertiesWithSystemOverrides(project));
    variables.put("project.version", project.getVersion());
    variables.put("project.groupId", project.getGroupId());
    variables.put("project.artifactId", project.getArtifactId());
    variables.put("project.build.directory", project.getBuild().getDirectory());
    variables.put("project.basedir", project.getBasedir().getAbsolutePath());
    variables.put("basedir", project.getBasedir().getAbsolutePath());
    return parser.read(inputStream, variables);
  }
```

?

Reply:
I didn't want to put all properties to the context, but let it be.